docs/PT_BR/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -442,7 +442,7 @@
     for implementar ou dimensionar seu HPCC System. As sugestões devem ser
     levadas em conta de acordo com as suas necessidades empresariais
     específicas.</para>
-
+<!-- COMMENTING FOR TEST - THIS WHOLE SECTION SHOULD BE REMOVED. 
```

Comment:
Should this be removed before merging?

Reply:
it should, but it needs to be replaced with the Portuguese content, to help ensure changes are made properly